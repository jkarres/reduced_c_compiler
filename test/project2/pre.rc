int gi = 0;

int gj = ++gi;

int gk = ++gi;

function : void main() {
  cout << 1 << " " << gi << endl;
  cout << 2 << " " << gj << endl;
  cout << 3 << " " << gk << endl;

  int li = 0;
  int lj = 10 + ++li;
  int lk = ++li + 10;

  cout << 4 << " " << li << endl;
  cout << 5 << " " << lj << endl;
  cout << 6 << " " << lk << endl;

}
