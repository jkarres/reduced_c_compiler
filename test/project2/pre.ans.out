1 2
2 1
3 2
4 2
5 11
6 12
